Dockerfile

```diff
@@ -29,11 +29,12 @@ RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get inst
 # We'll download rust manually to ensure signing looks good
 COPY .rust-key.gpg.ascii .rust-key.gpg.ascii
 COPY .rust-pgp-signature.asc rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz.asc
-RUN wget https://static.rust-lang.org/dist/rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz
-RUN cat .rust-key.gpg.ascii | gpg --import
-RUN gpg --verify rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz.asc rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz
-RUN tar -xf rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz
-RUN rust-1.46.0-x86_64-unknown-linux-gnu/install.sh
+RUN wget https://static.rust-lang.org/dist/rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz \
+  && cat .rust-key.gpg.ascii | gpg --import \
+  && gpg --verify rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz.asc rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz \
+  && tar -xf rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz \
+  && rm rust-1.46.0-x86_64-unknown-linux-gnu.tar.gz \
+  && rust-1.46.0-x86_64-unknown-linux-gnu/install.sh
```

Comment:
Could you remove the `&&`'s between the commands.
Otherwise, if a sub-command in the `RUN` command failed, then it may not be clear exactly which subcommand failed.